docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+# Cross Chain EVM Swap
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Understanding Lit PKPs as escrow
+- Lit Action
+- Implementing Mint/Grant/Burn with SDK
```

Comment:
m/g/b is currently broken, so this example doesn't work right now? Or was their a workaround you used?

Reply:
it's a workaround, so what m/g/b does is the following three steps
- mints you a pkp 
- adds lit action as a permitted action
- burns the pkp

usual m/g/b function on smart contracts should be doing these in a single transaction (as a single functional call to smart contract) whereas in this example I've individually called these through SDK and later instead of burning the pkp, transferred the pkp to itself in a single javascript function

This simulates a m/g/b scenario, any pkp minted with the same flow through js functions, works exact the same as minted by m/g/b function on contracts

the difference which makes the network failing is that m/g/b function burns the pkp nft (sending it to 0x00 address) whereas I'm sending it to pkp's address, in both the ways, pkp acts like escrow whereas network fails on reading pkp owner as 0x00

Reply:
gotcha, makes sense! Maybe we can add a callout box that makes this clear until the actual function is fixed

Reply:
sure, i'll add it

Reply:
does this look better?
https://github.com/LIT-Protocol/docs/pull/330/commits/30185e8fdd79532766306cf2736f33c5726e9f50#diff-0e36021eb2e9b13aa34e50ec77474d75d50bab8d18cba35c954841b81d9efee7R327